### PR TITLE
Update windows-instructions.md

### PR DESCRIPTION
**Pull request by ianhays**
Update the docs to include the new CMake requirement introduced in #5242

@ellismg

### CODE REVIEW

**Comment by mellinoe** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM

File: Documentation/building/windows-instructions.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ The following free downloads are compatible:
 
 Note: In order to build our C++ projects be sure to select "Programming Languages | Visual C++ | Common Tools for Visual C++ 2015" while installing VS 2015 (or modify your install to include it).
 
+[CMake](https://cmake.org/) is required to build the native libraries for Windows. To build these libraries cmake must be installed from [the CMake download page](https://cmake.org/download/) and added to your path.
```
Comment (stephentoub) [inline Documentation/building/windows-instructions.md]: Any restrictions on what version can/should be used?

Comment (ianhays) [inline Documentation/building/windows-instructions.md]: Not yet. We use 2.8.1.2 on Ubuntu (which uses all of the same stuff as on Windows) and 3.1 in fxkit. The latest (3.4.1) also works without issue. The version so far doesn't matter.

It's possible future updates to CMake might break something that we rely on though. If that's a worry then I can make the docs instruct to download 3.4.1, but it seems a bit... arbitrary.

Comment (stephentoub) [inline Documentation/building/windows-instructions.md]: Ok. Thanks.

**Pull request merged**